slash/fun.py

```diff
@@ -33,6 +33,19 @@ async def headsortails(self, interaction: Interaction):
         embed = nextcord.Embed(color=0x0DD91A, title=f"It is {hot}!")
         await interaction.response.send_message(embed=embed)
 
+    @slash(guild_ids=SLASH_GUILDS)
+    async def respect(
+        self,
+        interaction: Interaction,
+        text: str = SlashOption(description="What you respect", required=False),
+    ):
+        """Press F to pay respect"""
+        hearts = ("❤", "💛", "💚", "💙", "💜")
```

Comment:
More hearts?
```suggestion
        hearts = ("❤️", "🧡", "💛", "💚", "💙", "💜", "🖤", "🤎", "🤍")
```

Reply:
You could do that, but it might impact users on Apple, they might not have those emojis

Reply:
The emojis are directly copied from Discord, so all users should be able to see them

Reply:
Okay, again, just like my other pr, i will commit this soon (in the mobile app there is no commit directly button)